test/integration/ads_integration_test.cc

```diff
@@ -681,10 +595,10 @@ class AdsFailIntegrationTest : public AdsIntegrationBaseTest,
       ads_cluster->set_name("ads_cluster");
     });
     setUpstreamProtocol(FakeHttpConnection::Type::HTTP2);
-    AdsIntegrationBaseTest::initialize();
+    XdsIntegrationTestBase::initialize();
   }
 
-  FakeStreamPtr ads_stream_;
+  FakeStreamPtr xds_stream_;
```

Comment:
Nit: it would still be reasonable to call this `ads_stream_` in this file.

Reply:
That declaration was left over; it was supposed to have been moved into the new base class. I have removed this declaration. I would not have guessed that the compiler wouldn't complain about redeclaring a variable name in a subclass.